manifest.json

```diff
@@ -901,13 +901,13 @@
       "required": true
     },
     {
-      "projectID": 896591,
```

Comment:
You seem to be removing SerializationIsBad and the ToolBelt mod here, so uh, don't?

Reply:
blame toolbelt mod being added after this commit was made then
(I do not intend to remove these mods)